backend/controllers/refreshTokenController.js

```diff
@@ -0,0 +1,39 @@
+const {
+    getUserDataForRefresh,
+} = require('../database/actionsWithTables/userActions');
+const { sendError } = require('../util/sendError');
+
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+const handleRefreshToken = async (req, res) => {
+    try {
+        const cookies = req.cookies;
+        if (!cookies?.jwt) {
+            return res.sendStatus(401);
+        } 
+        const refreshToken = cookies.jwt;
+        const data = await getUserDataForRefresh(refreshToken);
+
+        jwt.verify(
+            refreshToken,
+            process.env.REFRESH_TOKEN_SECRET,
+            (error, decoded) => {
+                try {
+                    const accessToken = jwt.sign(
+                        { UserInfo: { login: decoded.login, is_admin: data.is_admin } },
+                        process.env.ACCESS_TOKEN_SECRET,
+                        { expiresIn: '30s' }
```

Comment:
It is better to make separate function in `authController` or `utils` and import here

Reply:
They work differently in authController - login: data.login and in refreshTokenController - login: decode.login. 
In authController we get it from dataBase and in refreshTokenController we get it from decoding refreshToken)